buildcontext/excludes.go

```diff
@@ -8,12 +8,12 @@ import (
 	"github.com/pkg/errors"
 )
 
-// EarthIgnoreFile is the name of the earth ignore file.
+// EarthIgnoreFile is the name of the earthly ignore file.
 const EarthIgnoreFile = ".earthignore"
```

Comment:
we're going to need some backwards compatibility if we want to rename this to `.earthlyignore`

Reply:
The other possible one is `Earthfile`... Perhaps let's leave them like this for now.